test_infra/test_cdn.py

```diff
@@ -8,7 +8,6 @@
 import pytest
 import requests
 
-MLS_URL = "https://location.services.mozilla.com/v1/country" "?key=ec4d0c4b-b9ac-4d72-9197-289160930e14"
```

Comment:
@pmac pretty sure this is safe to remove as we no longer use MLS, but just double checking with you?

Reply:
we probably can. this was a test to make sure our geo code was working properly. It assumes the MLS result is accurate, and asserts that our result is the same. Since we're removing the endpoint we're testing here let's just kill it.